Client/Application/AppDelegate.swift

```diff
@@ -13,64 +13,49 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // This is the easiest way to force a bootstrap that's guaranteed to happen on app launch
     private var appContainer: ServiceProvider = AppContainer.shared
-    var window: UIWindow?
-    var browserViewController: BrowserViewController!
+
     var rootViewController: UIViewController!
-    var tabManager: TabManager!
     var receivedURLs = [URL]()
     var orientationLock = UIInterfaceOrientationMask.all
-    lazy var themeManager: ThemeManager = DefaultThemeManager(appDelegate: self)
-    lazy var profile: Profile = BrowserProfile(localName: "profile",
-                                               syncDelegate: UIApplication.shared.syncDelegate)
+
     private let log = Logger.browserLogger
+
+    lazy var tabManager: TabManager = appContainer.resolve()
+    lazy var profile: Profile = appContainer.resolve()
+    lazy private var ratingPromptManager: RatingPromptManager = appContainer.resolve()
```

Comment:
Note: We're taking the container's instances and passing them down from this point. I'll look out for odd behavior but hopefully this doesn't introduce anything like that.